test/object-store/object.cpp

```diff
@@ -83,6 +85,73 @@ struct TestContext : CppContext {
     }
 };
 
+class TestContext2 {
```

Comment:
Something like `CreatePolicyRecorder` or `CreatePolicyRecordingContext` would describe what this is actually doing.